docs/fair-in-a-box.md

```diff
@@ -0,0 +1,22 @@
+# FAIR Data Point and EJP-RD Beacon API implementation
+
+Test in the local configuration folder (currently identical to global)
```

Comment:
I needed to copy the `.env` file to this folder in order for this to work, maybe that should be stated here.